README.md

```diff
@@ -1,2 +1,61 @@
 # structlog-sentry
-SentryProcessor for structlog
+
+| What          | Where                                         |
+| ------------- | --------------------------------------------- |
+| Documentation | <https://github.com/kiwicom/structlog-sentry> |
+| Maintainer    | @paveldedik                                   |
+
+Based on <https://gist.github.com/hynek/a1f3f92d57071ebc5b91>
+
+## Usage
+
+This module is intended to be used with `structlog` like this:
+
+```python
+import sentry_sdk
+import structlog
+from structlog_sentry import SentryProcessor
+
+
+sentry_sdk.init()  # pass dsn in argument or via SENTRY_DSN env variable
+
+structlog.configure(
+    processors=[
+        structlog.stdlib.add_log_level,  # required before SentryProcessor()
+        SentryProcessor(sentry_log_level=logging.ERROR),
+    ],
+    logger_factory=...,
+    wrapper_class=...,
+)
+
+
+log = structlog.get_logger()
+
+try:
+    1/0
+except ZeroDivisionError:
+    log.error()  # automatically logged in Sentry
```

Comment:
As we made it possible to log rich exceptions, please add two more examples. One for logging exception with message and details (extras). And one for logging just messages outside except block (with extras too). And use different levels in those examples (warning, error, critical), so it shows that it uses more Structlog features.